PyTorch/LanguageModeling/BERT/data/create_datasets_from_start.sh

```diff
@@ -21,9 +21,9 @@ python3 /workspace/bert/data/bertPrep.py --action sharding --dataset books_wiki_
 
 # Create HDF5 files Phase 1
 python3 /workspace/bert/data/bertPrep.py --action create_hdf5_files --dataset books_wiki_en_corpus --max_seq_length 128 \
- --max_predictions_per_seq 20 --vocab_file /workspace/bert/data/download/google_pretrained_weights/uncased_L-24_H-1024_A-16/vocab.txt
+ --max_predictions_per_seq 20 --vocab_file $BERT_PREP_WORKING_DIR/download/google_pretrained_weights/uncased_L-24_H-1024_A-16/vocab.txt --do_lower_case 1
 
 
 # Create HDF5 files Phase 2
 python3 /workspace/bert/data/bertPrep.py --action create_hdf5_files --dataset books_wiki_en_corpus --max_seq_length 512 \
- --max_predictions_per_seq 80 --vocab_file /workspace/bert/data/download/google_pretrained_weights/uncased_L-24_H-1024_A-16/vocab.txt
+ -max_predictions_per_seq 80 --vocab_file $BERT_PREP_WORKING_DIR/download/google_pretrained_weights/uncased_L-24_H-1024_A-16/vocab.txt --do_lower_case 1
```

Comment:
Aren’t you missing one “-“ in “-max_predictions_per_seq”, it should be “—max_ ...”?